packages/canon/.storybook/preview.tsx

```diff
@@ -1,8 +1,74 @@
-import type { Preview, ReactRenderer } from '@storybook/react';
-import { withThemeByDataAttribute } from '@storybook/addon-themes';
+import React, { useEffect } from 'react';
+import type { Decorator, Preview, ReactRenderer } from '@storybook/react';
+import { useGlobals } from '@storybook/preview-api';
+
+// Default Backstage theme
 import '../src/css/styles.css';
 
+// Custom themes
+import './themes/spotify.css';
+
+export const withThemes: Decorator = StoryFn => {
+  const [globals] = useGlobals();
+  const selectedTheme = globals.themeMode || 'light';
+  const selectedThemeName = globals.themeName || 'default';
+
+  useEffect(() => {
+    const htmlElement = document.documentElement;
+
+    // Remove any existing theme data attributes
+    htmlElement.removeAttribute('data-theme');
+    htmlElement.removeAttribute('data-theme-name');
+
+    // Add the selected theme data attribute
+    htmlElement.setAttribute('data-theme', selectedTheme);
+    htmlElement.setAttribute('data-theme-name', selectedThemeName);
+
+    // Cleanup on unmount
+    return () => {
+      htmlElement.removeAttribute('data-theme');
+      htmlElement.removeAttribute('data-theme-name');
+    };
+  }, [selectedTheme, selectedThemeName]);
+
+  return <StoryFn />;
+};
+
 const preview: Preview = {
+  globalTypes: {
+    themeMode: {
+      name: 'Theme Mode',
+      description: 'Global theme mode for components',
+      defaultValue: 'light',
+      toolbar: {
+        icon: 'circlehollow',
+        items: [
+          { value: 'light', icon: 'circlehollow', title: 'Light' },
+          { value: 'dark', icon: 'circle', title: 'Dark' },
+        ],
+        showName: true,
+        dynamicTitle: true,
+      },
+    },
+    themeName: {
+      name: 'Theme Name',
+      description: 'Global theme name for components',
+      defaultValue: 'default',
+      toolbar: {
+        icon: 'paintbrush',
+        items: [
+          { value: 'default', title: 'Default (Backstage)' },
+          { value: 'spotify', title: 'Spotify' },
```

Comment:
Wanna highlight this change. We thought it'd be useful to have a couple more themes available on the docs site, both to show the variation and be an inspiration for additional themes, but also for builders that want to target particular themes. This of course is open to anyone that wants to add their own theme down the line, although we'll probably avoid end user adopters due to clutter. We'll likely explore other ways to enable this too, but wanted to get started somewhere.